.github/workflows/push.yml

```
@@ -67,6 +68,8 @@ jobs:
       FEATURE_NAME: ${{ steps.git-branch-deploy.outputs.FEATURE_NAME }}
       PRE_CHECK: ${{ steps.should-run.outputs.PRE_CHECK }}
       PRE_RELEASE: ${{ steps.should-run.outputs.PRE_RELEASE }}
+      YARN_CACHE_FOLDER: ${{ steps.yarn-cache-dir-path.outputs.dir }}
```

actionlint (1.7.4)

property "yarn-cache-dir-path" is not defined in object type {git-branch: {conclusion: string; outcome: string; outputs: {string => string}}; git-branch-deploy: {conclusion: string; outcome: string; outputs: {string => string}}; should-run: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)